### PR TITLE
fix to add GLEW debug library

### PR DESCRIPTION
**Pull request by thoys**

### CODE REVIEW

File: cmake/modules/FindGLEW.cmake [hunk 1]
```diff
@@ -22,16 +22,16 @@ if (WIN32)
...
-  find_library(GLEW_LIBRARY_DEBUG glew32s PATH_SUFFIXES "lib/Debug/Win32" "lib" HINTS ${GLEW_SEARCH_DIRS})
+    find_library(GLEW_LIBRARY_RELEASE glew32s PATH_SUFFIXES "lib/Release/Win32" "lib" HINTS ${GLEW_SEARCH_DIRS})
+  find_library(GLEW_LIBRARY_DEBUG glew32sd PATH_SUFFIXES "lib/Debug/Win32" "lib" HINTS ${GLEW_SEARCH_DIRS})
   
   include(SelectLibraryConfigurations)
   select_library_configurations(GLEW)
 endif ()
 
-set(GLEW_LIBRARIES "${GLEW_LIBRARY}")
+set(GLEW_LIBRARIES ${GLEW_LIBRARY})
```
Comment (birarda) [inline cmake/modules/FindGLEW.cmake]: can you fix the indentation in this file? it looks off (you can especially tell if you click view)

Comment (thoys) [inline cmake/modules/FindGLEW.cmake]: there you go

**Comment by birarda** [comment]
waiting for build then will merge

**Comment by thoys** [comment]
seems to have build well, thanks for the help @birarda

**Pull request merged**